aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -189,12 +189,9 @@ Tensor sparse_coo_tensor(const Tensor& indices, const Tensor& values_, const Ten
       sparse_dim, dense_dim, computed_sizes, indices, values, values.options().layout(kSparse));
 }
 
-// NB: Got rid of the sizes == NULL case
-Tensor sparse_coo_tensor(const Tensor& indices, const Tensor& values_, ArrayRef<int64_t> size, const TensorOptions& options) {
+void _validate_sparse_coo_tensor_args(const Tensor& indices, const Tensor& values_, ArrayRef<int64_t> size) {
```

Comment:
I think you need a `TORCH_API` here, since this is the symbol that is "not found".

Reply:
There is `TORCH_API` in the header file. Besides, I believe `TORCH_API` doesn't do anything when building with gcc, as is the case on our dev machine.

Anyway, this was taking way more time than I considered worth it, so I abandoned this attempt and just left `_validate_sparse_coo_tensor_args` as a method on `torch`. I can try some more if you or another reviewer feel strongly about it.

Reply:
Don't feel too strongly, seems like the methods these are related to naturally go in `torch`. I was just hoping to give some pointers on how to make it work.